kDrive/AppDelegate.swift

```diff
@@ -463,9 +463,7 @@ final class AppDelegate: UIResponder, UIApplicationDelegate, AccountManagerDeleg
     // MARK: - Account manager delegate
 
     func currentAccountNeedsAuthentication() {
-        Task { @MainActor in
-            setRootViewController(SwitchUserViewController.instantiateInNavigationController())
-        }
+        Log.appDelegate("currentAccountNeedsAuthentication", level: .error)
```

Comment:
As this is no longer used, it should be removed.

Reply:
done